pages/linux/qm-showcmd.md

```diff
@@ -0,0 +1,16 @@
+# qm showcmd
+
+> Show command line which is used to start the VM (debug info).
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Show command line for a specific virtual machine:
+
+`qm showcmd {{vm_id}}`
+
+- Put each option on a new line to enhance human readability:
+
+`qm showcmd --pretty {{true}} {{vm_id}}`
+
+- Fetch config values from a given snapshot:
+
+`qm showcmd --snapshot {{string}} {{vm_id}}`
```

Comment:
What options can this `string` be?

Reply:
> What options can this `string` be?

I'm not completely sure but I am guessing, `string` should be `keys` of the configs for the vm.